### PR TITLE
Add fxa_users_services queries

### PR DESCRIPTION
**Pull request by jklukas**
Schedules new queries introduced in https://github.com/mozilla/bigquery-etl/pull/396

### CODE REVIEW

File: dags/fxa_events.py [hunk 1]
```diff
@@ -96,3 +96,46 @@
...
+
+    fxa_users_services_daily << fxa_auth_events
+    fxa_users_services_daily << fxa_auth_bounce_events
+    fxa_users_services_daily << fxa_content_events
+    fxa_users_services_daily << fxa_oauth_events
+
+    fxa_users_services_first_seen = bigquery_etl_query(
+        task_id='fxa_users_services_first_seen',
+        destination_table='moz-fx-data-shared-prod:telemetry_derived.fxa_users_services_first_seen_v1',
+        sql_file_path='sql/telemetry_derived/fxa_users_services_first_seen_v1/query.sql',
```
Comment (relud) [inline dags/fxa_events.py]: `init.sql` uses DDL that creates the table, and i'm not sure how that will behave in conjunction with `destination_table`, but `query.sql` doesn't exist
```suggestion
        sql_file_path='sql/telemetry_derived/fxa_users_services_first_seen_v1/init.sql',
```

Comment (jklukas) [reply dags/fxa_events.py]: I'm pretty sure a query will fail if it contains DDL and is configured with a destination table, so I will need to either switch init.sql to not use DDL or will need some new machinery here.

Comment (relud) [reply dags/fxa_events.py]: confirmed, it will fail

**Comment by jklukas** [comment]
Updated the machinery to allow DDL queries. Re-requesting review.

**Pull request merged**